config/locales/en.yml

```
@@ -77,6 +77,8 @@ en:
     proprietor:
       accounts:
         nav: Accounts
+      users:
+        nav: Users
```

I found another missing key: `proprietor.users.show.update`
```suggestion
        nav: Users
        show:
          update: Update
```